resources/function_help/json/$x

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "$x",
+  "name": "$vertex_Z_value",
   "type": "function",
-  "groups": ["GeometryGroup"],
-  "description": "Returns the x coordinate of the current point feature. If the feature is a multipoint feature, then the x-coordinate of the first point will be returned.",
-  "examples": [ { "expression":"$x", "returns":"42"}
+  "groups": ["Mesh"],
+  "description": "Returns the Z value of the current mesh vertex.",
+  "examples": [ { "expression":"$vertex_Z_value", "returns":"42"}
```

Comment:
@vcloarec Should this not be a new `vertex_z_value` (all lower case for the function?) file? Because I'm afraid you are  trying to remove the $x function (with some inconsistencies...)

Reply:
indeed, I had saved under the wring file...